application/Controllers/Checks.php

```diff
@@ -306,4 +306,9 @@ public function parser()
 		$this->parser = Services::parser();
 	}
 
+	public function blocks()
```

Comment:
The Checks controller is not something that is going to survive long and should not be the sole method of testing.